npm_tasks/explore.js

```diff
@@ -0,0 +1,34 @@
+// This is meant to provide a cross-platform way of doing the same thing
+// as "npm explore <name> -- <cmd>" (which doesn't work on Windows; see
+// https://github.com/npm/npm/issues/8932).
+//
+// Note that <cmd> is expected to be something extremely simple like
+// "npm install" or "git pull origin master". It shouldn't contain any
+// quoted arguments or other funky things.
+
+var fs = require('fs');
+var path = require('path');
+var execSync = require('child_process').execSync;
+
+var ROOT_DIR = path.resolve(__dirname, '..');
+
+function getModuleDir(moduleName) {
+  return fs.realpathSync(path.resolve(ROOT_DIR, 'node_modules', moduleName));
+}
+
+function explore(moduleName, cmd) {
+  execSync(cmd, { cwd: getModuleDir(moduleName), stdio: 'inherit' });
```

Comment:
is there a way to turn this into `spawn` instead of `exec`, instead? The problem with exec is that all the output from that step is lost (there is not stdout/stderr piping)